src/themes/material/templates/journal/article.html

```diff
@@ -254,13 +254,13 @@ <h4>
                       {% with article.get_doi_url as doi_url%}
                         <a class="waves-effect waves-light btn btn-small facebook-bg"
                             href="https://www.facebook.com/share.php?p[url]={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" 
-                            target="_blank"><i class="fa fa-facebook"></i></a>
+                            target="_blank" aria-label= {% trans "Share on Facebook" %}><i class="fa fa-facebook"></i></a>
                         <a class="waves-effect waves-light btn btn-small twitter-bg" 
                             href="https://twitter.com/intent/tweet?text={{ article.title }} {% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}"
-                            target="_blank"><i class="fa fa-twitter"></i></a>
+                            target="_blank" aria-label={% trans "Share on X" %}><i class="fa fa-twitter"></i></a>
                         <a class="waves-effect waves-light btn btn-small linkedin-bg"
                             href="https://www.linkedin.com/sharing/share-offsite?url={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}"
-                            target="_blank"><i class="fa fa-linkedin"></i></a></button>
+                            target="_blank" aria-label={% trans "Share on Linked In" %}><i class="fa fa-linkedin"></i></a></button>
```

Comment:
This has made me really think - the aria-label is to be read aloud, so I was writing it as it would be said, so I've had to check weather a screen reader will read LinkedIn intelligibly - and you are right, it reads it fine!